From 1f49ecbf3d2ab8003d37eb1c0454c5cfbe335ee5 Mon Sep 17 00:00:00 2001
From: Nicolas Roeser <nicolas.roeser@alumni.uni-ulm.de>
Date: Sun, 4 Aug 2024 14:34:44 +0200
Subject: [PATCH 158/201] md.4: replace wrong word

There is a wrong word in the md(4) man page, this commit corrects it.

Signed-off-by: Nicolas Roeser <nicolas.roeser@alumni.uni-ulm.de>
---
 md.4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/md.4 b/md.4
index 7a0bc7e6..7aef1577 100644
--- a/md.4
+++ b/md.4
@@ -224,7 +224,7 @@ option.  If you use this option to
 while running a newer kernel, the array will NOT assemble, but the
 metadata will be update so that it can be assembled on an older kernel.
 
-No that setting the layout to "unspecified" removes protections against
+Note that setting the layout to "unspecified" removes protections against
 this bug, and you must be sure that the kernel you use matches the
 layout of the array.
 
-- 
2.41.0

